src/nplinker/metabolomics/gnps/gnps_extractor.py

```diff
@@ -0,0 +1,93 @@
+from os import PathLike
+import os
+from pathlib import Path
+import zipfile
+
+from nplinker import utils
+from nplinker.metabolomics.gnps.gnps_format import GNPSFormat, gnps_format_from_archive
+
+
+class GNPSExtractor:
+    def __init__(self, filepath: str | PathLike, extract_path: str | PathLike):
+        """Class to handle extraction of spectra, molecular families and file mappings files from an archive located at `filepath` to the `extract_path`
```

Comment:
```suggestion
        """Extract spectra, molecular families and file mappings files from a GNPS archive.
```

Reply:
Please follow pep8 as much as possible:  [max line length for code is 79 and for docstring/comments is 72  ](https://peps.python.org/pep-0008/#maximum-line-length).

Also try to keep the first sentence of the docstring concise, usually limited to one line. If needed, more details could be provided in the 2nd paragraph.